db_json_util.py

```diff
@@ -52,8 +52,30 @@ def main() -> None:
         raise FileNotFoundError(f'Database file {args.file} not found')
 
     if args.api:
-        print('using api database')
-        # TODO: implement
+        # print('using api database')
+        if args.import_data:
+            if args.json_chains and args.json_rpc_urls:
+                # open the json file for chains table and send the data to the api
+                with open(path_chains, 'r', encoding='utf-8') as f:
+                    data_chains = json.load(f)
+                    for chain in data_chains:
+                        url_formatted = args.url + '/create_chain'
+                        response = requests.post(url_formatted, json=chain)
+                # print(response.status_code, response.text)
+                        if response.status_code == 201:
+                            print(f'added chain {chain["name"]}')
+                        else:
+                            print(response.text)
+                # open the json file for rpc_url table and send the data to the api
+                with open(path_urls, 'r', encoding='utf-8') as f:
```

Comment:
Like the other comment, let this block operate independently

```suggestion
            if args.json_rpc_urls:
                # open the json file for rpc_url table and send the data to the api
                with open(path_urls, 'r', encoding='utf-8') as f:
```